src/_community/rules.md

```diff
@@ -110,8 +138,9 @@ handle that discussion.
 
 ### Non-message mediums
 
-Do not change your avatar, nickname, username or custom/rich status to try to protest a moderation action, or to attack
-or harass any user.
+Do not change your avatar, nickname, username or custom/rich status to try to evade an infraction, or to attack or 
```

Comment:
This should probably be moved into a separate PR, since AFAIK there hasn't been any debate about this.